File: app/src/main/java/com/aero51/githubapp/db/model/RepoSearchResponse.java
package com.aero51.githubapp.db.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RepoSearchResponse {


        @SerializedName("total_count")
        public int total;
        public List<Repo> items;
        public int nextPage;

}

File: app/src/main/java/com/aero51/githubapp/db/model/RepoSearchResponse.kt
package com.aero51.githubapp.db.model

import com.google.gson.annotations.SerializedName

data class RepoSearchResponse (
        @SerializedName("total_count")
        val total: Int = 0,
        @JvmField
        val items: List<Repo>,
        val nextPage: Int
        ){


}